Using-browsec-for-securing-your-connection-to-fcc.md

```diff
@@ -0,0 +1,89 @@
+# Using Browsec for Securing your Connection to FreeCodeCamp.com
+
+## Why do I need an plugin or add-on?
+Lately some of the campers started facing strange problems, namely "Content Modification" by some Internet Service Providers (ISP), leading to the [FreeCodeCamp.com](http://freecodecamp.com) (FCC) website being broken sometimes.
+
+This came to light with some of the issues logged by campers :
+- [\#5999 : Indian ISP's injecting Ads breaks site](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/5999)
+- [\#6122 : Serious Problem using the FCC IDE, console showing error...](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/6122)
+- [\#6381 : Unable to view the Code editor in the browser](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/6381)
+
+## TL;DR : What's Wrong, again?
+
+Well, for some campers, this is a classic case of [man in the middle attack](https://en.wikipedia.org/wiki/Man-in-the-middle_attack). 
+
+For instance, as seen in the issue \#5999, the ISP of some campers from India is deliberately injecting Ads into the user's visited websites that's causing problems.
+
+While, in other cases, ISPs sometimes use a similar strategy to cache the websites visited by consumers, so that they can serve the cached content to other consumers visiting the same websites, this saves them bandwidth costs, while making websites available faster.
+
+But when this is not done correctly (or is done malaciously as in the first case), this modifies essential content for websites, and prevents them to run seemlessly.
+
+## What's the fix, then?
+
+Simple, we need to anonymize ourselves i.e connect to the FCC website, by encrypting our traffic so it cannot be modified during transmission. We need to by-pass the ISP, or the firewall caches, whatsoever.
```

Comment:
Although that makes it redundant.  Why do we need this second sentence at all?

I'm not sure that we're really "anonymizing" our traffic as much as encrypting it so it cannot be intercepted and modified.